app/templates/services/list_services.html

```diff
@@ -30,35 +29,34 @@ <h1 class="govuk-heading-l">Your {{ framework.name }} services</h1>
     </div>
   </div>
 
-  {% set empty_message %}
-    You don’t have any {{ framework.name }} services on the Digital Marketplace
-  {% endset %}
-  {% call(item) summary.list_table(
-    services,
-    caption='Current services',
-    field_headings=[
-      'Name',
-      'Lot',
-      summary.hidden_field_heading("Status")
-    ],
-    field_headings_visible=True,
-    empty_message=empty_message
-  ) %}
-    {% call summary.row() %}
-      {{ summary.service_link(
-          item.serviceName or item.lotName,
-          url_for('.edit_service', framework_slug=item.frameworkSlug, service_id=item.id)
-      ) }}
+  {% if services %}
+    {% set ns = namespace(rows = []) %}
+    {% for service in services %}
+      {% set service_link %}
+        <a class="govuk-link" href="{{ url_for('.edit_service', framework_slug=service['frameworkSlug'], service_id=service['id']) }}">{{ service.serviceName }}</a>
+      {% endset %}
 
-      {{ summary.text(item.lotName or item.lot) }}
+      {% set row = ns.rows.append(
+        [
+            {"html": service_link},
+            {"text": service.get("lotName") or service["lot"]},
+            {"text": "Live" if service["status"] == "published" else "Removed"},
+        ]
+      ) %}
+    {% endfor %}
 
-      {% call summary.field(action=True) %}
-        {% if item.status == "published" %}
-          Live
-        {% else %}
-          Removed
-        {% endif %}
-      {% endcall %}
-    {% endcall %}
-  {% endcall %}
+    {{ govukTable({
+      'head': [
+        {'text': "Name", "classes": "govuk-!-width-one-half "},
+        {'text': "Lot"},
+        {
+          'text': "",
```

Comment:
Here, I'm just copying what came before. Should we replace this with `"Status"` instead for accessibility?

Reply:
Good call. "Lot" is not necessarily a great  heading either - I got the sense that people who regularly work with DOS would know that really well, but other users might not. We decided to keep "Lot" when this came up last.